### ISSUE TITLE
graph grid

### ISSUE THREAD

**Issue by rsklyar** [rrag/react-stockcharts#20]
Could you provide additional element for all type of charts - stock grid?

Within the graph a grid of lines may appear to aid in the visual alignment of data. The grid can be enhanced by visually emphasizing the lines at regular or significant graduations. The emphasizing lines are then called major grid lines and the remainder are minor grid lines.

**Comment by rrag** [comment]
how would I differentiate between major and minor?

I was trying to show how this can be done right now for all ticks, I may have identified a defect, in how the `innerTickSize` is used for displaying the text on the tick.

**Comment by rsklyar** [comment]
If the grid is displayed as lines rather than dots, darker lines called major grid lines display at intervals. When working in decimal, major grid lines could control by frequency of grid lines in the expression ( max Xaxis - min(Xaxis)) / n (where n - frequency)

![ill_grid_002](https://cloud.githubusercontent.com/assets/6270599/10295892/963e8efa-6bcc-11e5-9e01-da4286d8b091.png)

**Comment by rsklyar** [comment]
But, i also recommend display minor grid lines using dots. 
<img width="294" alt="screen shot 2015-10-06 at 01 50 59" src="https://cloud.githubusercontent.com/assets/6270599/10295921/c4087134-6bcc-11e5-8b1e-46a56ec2f3b0.png">

or, you could use simple css transform (class grid), using this example

http://www.d3noob.org/2013/01/adding-grid-lines-to-d3js-graph.html

**Comment by rrag** [comment]
Began to work on the minor, but soon I made up my mind that it is not adding any value.

Added major grid only, see example here
http://rrag.github.io/react-stockcharts/documentation.html#/indicators_stochastic_oscillator

I am not in favor of using dots as it is a hassle to draw that on canvas. I want to get svg and canvas produce the same exact chart visually.

react-stockcharts@0.2.5 available on npm with this and a few other minor defect fixes. look at [change_log](http://rrag.github.io/react-stockcharts/documentation.html#/change_log) for full list
